dashboard/app/controllers/ai_diff_controller.rb

```diff
@@ -0,0 +1,66 @@
+class AiDiffController < ApplicationController
+  include AiDiffBedrockHelper
+  # authorize_resource class: false
```

Comment:
looks like you already tried using CanCan and found some docs! :-) with the other changes below, you should be able to uncomment this line now.